ci/jenkins/ssh-config

```diff
@@ -0,0 +1,8 @@
+Host CONTROLPLANENODE
```

Comment:
Is this the best place for this file as well? I think it belongs under `ci/cluster-api/` too.

Reply:
This file is for e2e test. It used to be unavailable for other developers.
https://github.com/vmware-tanzu/antrea/blob/master/ci/jenkins/jobs/macros.yaml#L101

Our new jenkins job script will process these templates. I will make a PR of all updated jenkins job code after testing finished.